tests/unit/amo/reducers/test_reviews.js

```diff
@@ -296,43 +296,124 @@ describe(__filename, () => {
   });
 
   describe('clearAddonReviews', () => {
-    it('clears add-on reviews', () => {
+    const loadReviewDataIntoState = ({
+      addonId,
+      addonSlug,
+      grouping,
+      startState,
+      reviewId,
+      userId,
+    }) => {
+      const review = {
+        ...fakeReview,
+        id: reviewId,
+        addon: {
+          id: addonId,
+          slug: addonSlug,
+        },
+        user: { id: userId },
+      };
+
+      let state;
+
+      // Initialize values into the byId, byAddon, byUserId and groupedRatings buckets.
+      state = reviewsReducer(startState, setReview(review));
+      expect(state.byId[reviewId].addonId).toEqual(addonId);
```

Comment:
I know it is convenient to add expectations in a function helper, but then it gets executed all the time and it is not easy to see when it breaks (since we usually find expectations in the test cases, not in helper functions). I think we should either not have expectations or they should be in the test cases.